articles/implementing-network-address-translation/index.md

```diff
@@ -0,0 +1,205 @@
+In this situation, a NAT pool is not created, but an outside interface used for the translation is used in place of the NAT pool in step 3 above.
+1. Create an ACL using the *access-list **1** permit **address wildcard mask** 
+2. PAT is enabled using the  **ip nat inside source list ACL number interface interface type/number overload**
+*The interface used for this is the outside interface, and it is configured as the single public address assigned to the organization by an ISP*.
+
+3. Use the **ip nat inside** interface command to enable the inside interface for Nat translation
+4. Use the **ip nat outside** interface command to enable the outside interface for NAT translation.
+
+### Configuring PAT with single single
+An organization was assigned one public address, 200.100.100.1, and wants to allow its internal hosts, in the private network 172.31.1.0  255.255.255.0 to reach the internet using PAT
+![PAT topology](/engineering-education/implementing-network-address-traslation/pattopo.jpg)
+To configure PAT for the topology above, the fowolling steps are applied
+-  Create an ace list that will specify which private addresses are allowed to be translated using the **access-list 1 permit 172.31.1.0 0.0.0.255**
+- Bind the access list and the outside interface together using **the ip inside source list 1 interface s0/0/0 overload**.
+- Enter serial 0/0/0/ interface command mode to identify it as the outside interface relative to NAT using the: ip nat outside command.
+- Enter gigabitethernet 0/0 using the interface gigabitethernet g0/0 command and identify it as the inside interface relative to NAT using the ip nat inside command
+ ![PAT with single address configuration on a CISCO router](/engineering-education/implementing-network-address-translation/patsincon.jpg)
+
+### Conclusion
+The introduction of private and public addressing and Network Address Translation helped slow down the exhaustion of Internet Protocol version 4 (IPV4).
+But implementing NAT also had an unintended consequences, which is providing a layer of security to the internal network by hiding their internal IP address.
+To summarized
+The reader learned:
+- What network address translation is
+- Different types of NAT and how to configure them on a CISCO router.
+
+### Activity files 
+For a better understaing of Network address traslation, the following files are provided for practice.
+- [Staic NAT packet tracer activity files](https://drive.google.com/file/d/1GO_LIIUx_qBHQRMWuHQBpzIepQpUM85d/view?usp=sharing)
+- [Dynamic packet tracer activity file](https://drive.google.com/file/d/1pwYgVJVSVQStPBy5ftsFtXwIrSq24pEK/view?usp=sharing)
+- [PAT with multiple addresses activity file](https://drive.google.com/file/d/1Xf2qxUJ-6X74vsJY4HZ8Z9tZW2C3hT7j/view?usp=sharing)
+- [PAT with single address activity file](https://drive.google.com/file/d/1_pBXkLEFshdthJuZHNwLgHOvoapybVEL/view?usp=sharing)
```

Comment:
You have not made the Google Drive file/folder public. Alternatively, you can put them all in a GitHub repo.